x/stakeibc/keeper/transfer.go

```diff
@@ -0,0 +1,102 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	ibctypes "github.com/cosmos/ibc-go/v7/modules/apps/transfer/types"
+	clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+
+	"github.com/Stride-Labs/stride/v14/x/stakeibc/types"
+)
+
+// Transfers tokens from the community pool deposit ICA account to the host zone holding address for that pool
+func (k Keeper) IBCTransferCommunityPoolTokens(ctx sdk.Context, token sdk.Coin, communityPoolHostZone types.HostZone, shouldLiquidStake bool) error {
```

Comment:
nit: I respect the descriptive variable names 🤝 , but for these functions I think it might make sense to change `communityPoolHostZone` → `hostZone`, and `communityPoolHostZoneId` to `hostZoneId` since there's only one host zone in question in the function and it makes the code slightly more concise